codalab/worker_manager/aws_batch_worker_manager.py

```diff
@@ -60,6 +69,17 @@ def add_arguments_to_subparser(subparser):
 
     def __init__(self, args):
         super().__init__(args)
+        # We import this lazily, so a user doesn't have to install boto3 unless
+        # they absolutely want to run the AWS worker manager (versus if it's incidentally)
+        # imported by other code (e.g., to access AWSBatchWorkerManager.DESCRIPTION , as done
+        # in codalab/worker_manager/main.py ).
+        try:
+            import boto3
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError(
+                "Running the AWS worker manager requires the boto3 module.\n"
+                f"Please run: pip install {get_boto3_pip_version()}"
```

Comment:
I feel like using `get_boto3_pip_version` is adding a lot of complexity, and I don't think we have tests that catch this (so, if we rename requirements-server.txt, the AWSBatchWorkerManager will silently fail). Moreover, it appears that boto3 hasn't ever released any breaking changes. Could we just say "Please run: pip install boto3" instead?

Reply:
good idea, i'll make this change